File: server/src/main/java/backend/SabotenApplication.java
package backend;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@EnableJpaAuditing
@SpringBootApplication
public class SabotenApplication {
    public static void main(String[] args) {
        SpringApplication.run(SabotenApplication.class, args);
    }
}

File: server/src/main/kotlin/backend/SabotenApplication.kt
package backend

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import common.entities.User

@SpringBootApplication
class SabotenApplication

fun main(args: Array<String>) {
    val user = User(0, "", "")
    runApplication<SabotenApplication>(*args)
}

